### PR TITLE
refactor(graphics): add missing virtual keyword on events interfaces

### PR DESCRIPTION
**Pull request by flavien-chenu**
## Changes made

I added the keyword `virtual` on graphics events interfaces to allow multiple inheritance in implementations. **It will change nothing for work that has already been done. 🎉**

### CODE REVIEW

File: a.out
Comment (TekMath) [inline a.out:1]: OOOUUUULA

**Review by TekMath — CHANGES_REQUESTED** [review]

File: main.cpp
Comment (TekMath) [inline main.cpp:1]: Not sure about this file

**Review by TekMath — CHANGES_REQUESTED** [review]

File: a.out
Comment (flavien-chenu) [reply a.out:1]: OOOOOOOOOOOOOOOOUUUUUUULLLLLLLA oui je viens de voir

**Review by TekMath — APPROVED** [review]
Nice

**Review by Yann-Masson — APPROVED** [review]

**Review by sdragos1 — APPROVED** [review]

**Pull request merged**